conn.go

```diff
@@ -182,12 +183,19 @@ var (
 )
 
 func newMaskKey() [4]byte {
-	n := rand.Uint32()
-	return [4]byte{byte(n), byte(n >> 8), byte(n >> 16), byte(n >> 24)}
+	/*n := mr.Uint32()
```

Comment:
Simplify to:

    var k [4]byte
    rand.Read(k[:]) // where rand is crypto/rand
    return k